src/timecop/core_ext/time.cr

```diff
@@ -9,6 +9,13 @@ struct Time
     @@mock = true
   end
 
+  def self.monotonic_without_mock_time
+    @@mock = false
+    monotonic
+  ensure
+    @@mock = true
+  end
```

Comment:
This may not work well in concurrent contexts.

Reply:
True, but it is following the paradigm at play in the repo.